package.json

```diff
@@ -6,7 +6,7 @@
     "build": "NODE_ENV=production node ./scripts/make.js",
     "dev": "NODE_ENV=development node ./scripts/make.js",
     "release": "node ./scripts/publish.js",
-    "prettier": "prettier-eslint --config ./.prettierrc --write \"./src/**/*.js\"",
+    "prettier": "prettier-eslint --config ./.prettierrc --write \"./src/**/*.{js,jsx}\"",
     "flow": "flow check",
     "eslint": "eslint ./src -c ./.eslintrc.yml --cache --cache-location ./.cache/.eslintcache",
     "test": "jest --config=./jest.config.js --forceExit --detectOpenHandles --no-cache",
```

Comment:
I think we should add `react` as a `peerDependency`, to indicate that while React is not strictly required to use Watermelon, there are APIs exposed that rely on a recent version of React

Reply:
I've added react@^16.5.0 since React Native currently depends on the ^16.6 alpha, seemed like the safest option, let me know if that's not right.